src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/settingsComponents/ShowAnswerCard.jsx

```diff
@@ -13,12 +13,12 @@ const ShowAnswerCard = ({
   showAnswer,
   updateSettings,
   defaultValue,
-  // redux
-  studioEndpointUrl,
-  learningContextId,
-  isLibrary,
+  intl,
 }) => {
-  const intl = useIntl();
```

Comment:
This was correct - please don't replace `useIntl` with `injectIntl`; the latter is deprecated.

It should show this if you're using an IDE:
<img width="324" height="36" alt="Screenshot 2025-08-22 at 3 06 30 PM" src="https://github.com/user-attachments/assets/cf566639-53b5-4ec9-8c40-a2846be95f03" />

Reply:
^ This hasn't been fixed yet.